portal/models/i18n.py

```diff
@@ -19,12 +21,15 @@ def get_db_strings():
     elements = defaultdict(set)
     for entry in AppText.query:
         if entry.custom_text:
-            elements['"{}"'.format(entry.custom_text)].add("apptext: " + entry.name)
+            ct = re.sub('"', r'\\"', entry.custom_text)
```

Comment:
Sorry, that comment was from before you refactored it into smaller functions. No need to make any changes :)